RFS/src/main/java/org/opensearch/migrations/bulkload/common/InvalidResponse.java

```diff
@@ -85,4 +86,36 @@ private static Map.Entry<String, String> getUnknownSetting(JsonNode json) {
             return Map.entry(entry.getKey().asText(), matcher.group(1));
         }).orElse(null);
     }
+
+    /** Awareness attribute exceptions (when the replica count doesn't match the number of zones) present slightly differently
+     in different versions. The message is the same (`"Validation Failed: 1: expected total copies needs to be a multiple of total awareness attributes [3];"`,
+     for instance), but the type is different (either `validation_exception` or `illegal_argument_exception`). For this reason,
+     we're matching based on a regex against the message instead of also checking the type.
+    **/
+    public Optional<String> containsAwarenessAttributeException() {
```

Comment:
Does the message need to be parsed and returned by this function, that would aid the user if we displayed it to them?  If not, this function could be reduced to a return a `boolean` value.